src/driver/hermes_mod.c

```diff
@@ -55,24 +55,25 @@ static const struct pci_device_id pci_ids[] = {
 };
 MODULE_DEVICE_TABLE(pci, pci_ids);
 
-static void irq_teardown(struct xdma_dev *xdev)
+static void irq_teardown(struct hermes_pci_dev *hpdev)
```

Comment:
I got to the end of the patch set and I still don't see these being used for anything.

Why not open code these now, then create a new helper when you actually have something to add to it?

Besides that this seems like some decent cleanup.

Reply:
Sure, I can move some of these commits to #23 where they'll be used.